src/components/widgets/BlackBorderDropdown.tsx

```diff
@@ -140,6 +154,37 @@ const SaveBlackBorderDropdown: React.FunctionComponent<
       ))}
     </Select>
   )
+
+  const searchableDropdown = (
+    <Box>
+      <Box style={{fontSize: '14px', fontFamily: poppinsFont}}>
+        Participant Time Zone
```

Comment:
This is a generic control. It should not have things specific to a certain usecase